modules/products/jira/helm.tf

```diff
@@ -52,10 +52,9 @@ resource "helm_release" "jira" {
         sharedHome = {
           customVolume = {
             persistentVolumeClaim = {
-              claimName = var.pvc_claim_name
+              claimName = kubernetes_persistent_volume_claim.share-home-pvc.metadata.0.name
             }
           }
-          subPath = "${local.product_name}-${random_string.random.result}"
```

Comment:
Not sure, I feel we kind of need to keep subPath at a point. Let's first test running Jira and Bitbucket together in the same environment while both using `nfs` for shared home to make sure if we have two nfs instances, they don't share the volume. I tested a couple of days ago and I faced some problem. I had not enough time to investigate more. It may was some other issue.

Reply:
If we have a different NFS server for each product (decision we can revisit later), I don't think it is a problem as the volumes are separate.

We still need to retest the products after they are all on NFS servers.

Reply:
Agree... I found a bug in NFS and it may caused the problem. However let's test it again when PR#211 merged.